src/layer/FeatureGroup.js

```diff
@@ -22,7 +22,7 @@ import {LatLngBounds} from '../geo/LatLngBounds';
  * 	.addTo(map);
  * ```
  */
-
+console.log(LayerGroup);

```

Comment:
console.log(LayerGroup);

Reply:
Oh Thanks! Will remove that. But as said above still WIP. So there might be some console.log or debugger calls in the code...